trunk/SUAVE/Attributes/Solids/Aluminum.py

```diff
@@ -0,0 +1,30 @@
+# Aluminum.py
+#
+# Created: Jul 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+## @ingroup Attributes-Solids
+
+from Solid import solid
+from SUAVE.Core import Data, Units
+
+#-------------------------------------------------------------------------------
+# Aluminum 6061-T6 Solid Class
+#-------------------------------------------------------------------------------
+
+class aluminum(solid):
+
+    """ Physical Constants Specific to Aluminum 6061-T6"""
+
+    def __defaults__(self):
+
+        self.ultimate_tensile_strength        = 310e6    *Units.Pa
```

Comment:
periods

Reply:
Usage of exponential notation also forces float data type.